### PR TITLE
updating caption

### PR DESCRIPTION
**Pull request by CraigShutko**
This pull request is totally basic.

### CODE REVIEW

**Comment by CraigShutko** [comment]
Fixes some bugs #3

File: _slides/craigshutko.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (juhewitt) [inline _slides/craigshutko.md:1]:
``` suggestion
---
layout: slide
title: "Welcome Craig's slide!"
---

**Review by juhewitt — APPROVED** [review]

File: _slides/craigshutko.md (hunk 1 above)
Comment (NoorChasib) [inline _slides/craigshutko.md:1]: pretty small commit

Comment (NoorChasib) [inline _slides/craigshutko.md:1]:
```suggestion
```

Meow meow meow




crlf




```
```

**Review by NoorChasib — APPROVED** [review]

**Pull request merged**